Chap_Terms.tex

```diff
@@ -105,9 +96,9 @@ \section{Naming Conventions}
 \item The \code{pmix_} prefix is used to denote function pointer and type definitions.
```

Comment:
(Note from the Reading in 2Q 2020 ASC Meeting)
 * In the first two bullet points add the `_` to the `\textbf{\code{PMIX_}}` and `\code{pmix_}`